portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSet.testcase

```diff
@@ -0,0 +1,60 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-data-set-taglib";
```

Comment:
Looks like the widget wasn't added. 

property osgi.modules.includes = "frontend-data-set-sample-web";